sources/packages/backend/apps/db-migrations/src/sql/StudentAssessments/Add-student-assessment-status-cols.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE
+  sims.student_assessments
+ADD
+  COLUMN student_assessment_status sims.student_assessment_status NOT NULL DEFAULT 'Created';
+
+COMMENT ON COLUMN sims.student_assessments.student_assessment_status IS 'Student assessment status from its creation till the workflow calculations are finalized or the workflow is cancelled.';
+
+ALTER TABLE
+  sims.student_assessments
+ADD
+  COLUMN student_assessment_status_updated_on TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW();
+
+COMMENT ON COLUMN sims.student_assessments.student_assessment_status_updated_on IS 'Date and time when the student_assessment_status column was set.';
```

Comment:
column was `set or updated`?

Reply:
I will change it to "Date and time when the student_assessment_status column was updated."